haystack/nodes/answer_generator/transformers.py

```diff
@@ -97,6 +98,9 @@ def __init__(
         :param embed_title: Embedded the title of passage while generating embedding
         :param prefix: The prefix used by the generator's tokenizer.
         :param use_gpu: Whether to use GPU. Falls back on CPU if no GPU is available.
+        :param progress_bar: Whether to show a progress bar, defaults to True.
+        :param devices: List of GPU devices to limit inference to certain GPUs and not use all available ones (e.g. [torch.device('cuda:0')]).
```

Comment:
Can we change to: 
List of GPU devices you want to limit inference to so as not to use all available GPUs. Example: [torch.device(cuda:0)].